Libplanet/AssemblyInfo.cs

```diff
@@ -1,6 +1,7 @@
 #nullable disable
 using System.Runtime.CompilerServices;
 
-[assembly: InternalsVisibleTo("Libplanet.Tests")]
 [assembly: InternalsVisibleTo("Libplanet.Extensions.Cocona.Tests")]
-[assembly: InternalsVisibleTo("Libplanet.Benchmarks")]
+[assembly: InternalsVisibleTo("Libplanet.Net")]
```

Comment:
It will be great if we can remove this and make `Libplanet` internals used in `Libplanet.Net` public.

Reply:
I filed a new issue to address this: <https://github.com/planetarium/libplanet/issues/1762>.  Thanks!